src/system/tests/TestTLVPacketBufferBackingStore.cpp

```diff
@@ -35,6 +35,18 @@ using namespace ::chip;
 
 namespace {
 
+void WriteUntilRemainingLessThan(nlTestSuite * inSuite, PacketBufferTLVWriter & writer, const uint32_t & remainingSize)
```

Comment:
`const &` on a uint32 feels odd ... a pointer will likely be at least 32 bit so this likely does not save call stack size.